lib/jmap/core/capability/websocket_capability.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:jmap_dart_client/jmap/core/capability/capability_properties.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'websocket_capability.g.dart';
+
+@JsonSerializable()
+class WebSocketCapability extends CapabilityProperties {
+  final bool supportsPush;
+  final Uri url;
```

Comment:
:+1:

If you need link to the spec: https://datatracker.ietf.org/doc/html/rfc8887